pkg/pb/contact.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package contact;
+
+import "gogo.proto";
+import "node.proto";
+
+service Contact {
+    rpc PingNode(ContactPingRequest) returns (ContactPingResponse);
+}
+
+service Node {
+    rpc Checkin(CheckinRequest) returns (CheckinResponse);
+}
+
+message CheckinRequest {
+    node.NodeAddress sender = 1;
+    bytes sender_nodeid = 2 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
+    node.NodeCapacity capacity = 3;
+    node.NodeOperator operator = 4;
+}
+
+message CheckinResponse {
+    bool ping_node_success = 1;
+    string error_message = 2;
```

Comment:
Do we need this? If there's an error, it will be returned separately from the response

Reply:
the error_message is details about ping_node_success if it is false

Reply:
i think i will change it to `ping_error_message`